src/patternfly/components/AppLauncher/themes/dark/app-launcher.scss

```diff
@@ -10,6 +10,6 @@
     --pf-c-app-launcher__menu-item--m-favorite__menu-item--m-action--Color: var(--pf-global--palette--gold-300);
     --pf-c-app-launcher__menu-item--m-favorite__menu-item--m-action--hover--Color: var(--pf-global--palette--gold-100);
     --pf-c-app-launcher__group-title--FontSize: var(--pf-global--FontSize--xs);
-    --pf-c-app-launcher__group-title--FontWeight: var(--pf-global--FontWeight--bold);
+    --pf-c-app-launcher__group-title--FontWeight: var(--pf-global--FontWeight--normal);
```

Comment:
This applies to the updates in all of these components - since the light theme will set these as default, dark theme will inherit them, so they don't need to be defined in the dark theme stylesheet.

Reply:
Can I also remove `--pf-c-dropdown__group-title--FontSize: var(--pf-global--FontSize--xs);` since light theme will pick up as well?